commands/commands.go

```diff
@@ -253,6 +254,19 @@ func usage(cmd *cobra.Command) error {
 	return nil
 }
 
+// isCommandEnabled returns whether the environment variable GITLFS<cmd>ENABLED
+// is equal to 1, returning false by default if the enviornment variable is not
+// specified.
+//
+// This function call should only guard commands that do not yet have stable
+// APIs or solid server implementations.
+func isCommandEnabled(cmd string) bool {
+	env := os.Getenv(fmt.Sprintf("GITLFS%sENABLED", strings.ToUpper(cmd)))
+	v, _ := strconv.Atoi(env)
```

Comment:
As discussed, went for [`config.Configuration.GetenvBool`](https://godoc.org/github.com/github/git-lfs/config#Configuration.GetenvBool) in https://github.com/github/git-lfs/pull/1386/commits/ab9842c3f97113e8c304b32f4040fbb2284dd9a5. 😄